data-prepper-plugins/http-sink/src/test/java/org/opensearch/dataprepper/plugins/sink/configuration/HttpSinkConfigurationTest.java

```diff
@@ -23,14 +23,8 @@
 
 public class HttpSinkConfigurationTest {
 
-    private static final String SINK_YAML = "        urls:\n" +
-            "        - url: \"https://httpbin.org/post\"\n" +
-            "          workers: 1\n" +
-            "          proxy: test\n" +
-            "          codec:\n" +
-            "            ndjson:\n" +
-            "          http_method: \"POST\"\n" +
-            "          auth_type: \"http_basic\"\n" +
+    private static final String SINK_YAML =
+            "        url: \"https://httpbin.org/post\"\n" +
```

Comment:
Please test with a localhost address instead of this webpage.

Reply:
Thanks for the review @dlvenable , changed the url to localhost:8080.